engine-tests/out/test/resources/gameManifest.json

```diff
@@ -10,7 +10,7 @@
       "title": "main",
       "seed": "naqrhaKV1lUofG7PLIJcOfDlUiPs3Jmp",
       "time": 25538256,
-      "worldGenerator": "Core:heightMap"
+      "worldGenerator": "BuilderSampleGameplay:BuilderSampleWorld"
```

Comment:
I don't know what this `gameManifest`.json` is used for, to be honest. I just replaced this value with the builder sample world as that is the only one still living in the engine repo....

Does anybody know whether the file is still used?

Reply:
Good question. Being that it was set to a generator that doesn't work ... probably either not being used or that particular value doesn't matter anyway :-)

We don't generally delete unit tests though, so maybe something is hiding somewhere and still at least trying to use this 🤔 Unless maybe it got moved out as part of the Record & Replay system, or for the MTE?

Reply:
If a unit test is using this and the value does not matter, we should make that clearer by using something like `Gooey:DummyWorld` or similar. If it is not used, we should delete it... 
I can try to delete it and see if anything breaks in the tests...